replication.go

```diff
@@ -725,6 +690,51 @@ func (cm *ContentManager) startup() error {
 	return cm.queueAllContent()
 }
 
+func (cm *ContentManager) reBuildStagingZones() error {
+	log.Info("rebuilding staging zones.......")
+
+	var stages []Content
+	if err := cm.DB.Find(&stages, "not active and pinning and aggregate").Error; err != nil {
+		return err
+	}
+
+	zones := make(map[uint][]*contentStagingZone)
+	for _, c := range stages {
+		z := &contentStagingZone{
+			ZoneOpened: c.CreatedAt,
+			CloseTime:  c.CreatedAt.Add(maxStagingZoneLifetime),
+			MinSize:    minStagingZoneSizeLimit,
+			MaxSize:    maxStagingZoneSizeLimit,
+			MaxItems:   maxBucketItems,
+			User:       c.UserID,
+			ContID:     c.ID,
+			Location:   c.Location,
+		}
+
+		minClose := time.Now().Add(stagingZoneKeepalive)
+		if z.CloseTime.Before(minClose) {
+			z.CloseTime = minClose
+		}
+
+		var inzone []Content
+		if err := cm.DB.Find(&inzone, "aggregated_in = ?", c.ID).Error; err != nil {
+			return err
+		}
+
+		z.Contents = inzone
+
+		for _, zc := range inzone {
+			// TODO: do some sanity checking that we havent messed up and added
```

Comment:
Are we implementing this on this PR? I'm not entirely sure how you will quantify a limit per `Zone`. I'm guessing this will be part of the "Fair Usage distribution?"

Reply:
No, this was just a lift and drop code - no new code was added (nothing in this PR has anything to do with fair usage)